src/styles/tablet.css

```
@@ -1,3 +1,29 @@
 body {
 	color: blue;
 }
+
+footer {
```

Instead of using a tag selector, use a class selector
```suggestion
.footer {
```